src/BLEBondStore.cpp

```diff
@@ -1,10 +1,14 @@
 // Copyright (c) Sandeep Mistry. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+// Contributors:
+//    Bosch Software Innovations GmbH - Please refer to git log
+
 
 #ifdef __AVR__
   #include <avr/eeprom.h>
 #elif defined(NRF51) || defined(NRF52) || defined(__RFduino__)
-  // nothing extra needed
+  #include "nrf_soc.h"
```

Comment:
We need to make sure that all boards have this file. I'll look into setting up Travis CI to test compiling of the library on various boards.